### ISSUE TITLE
Does GA Still Working Now?

### ISSUE THREAD

**Issue by mdestafadilah** [pagestudiocms/cms#81]
I just try your stable version, Google Analytics is not working yet?

**Comment by cosmomathieu** [comment]
What error do you get? Did you try what we talked about here #73?

**Comment by mdestafadilah** [comment]
Yes, i try ... in view settings in google analytics ... i choose **http** not **https**   but wait untul 24Hours after change .. i will post later
